### PR TITLE
Update readme after architecture change

### PR DESCRIPTION
**Pull request by Tbaut**
This part slipped my mind before merging https://github.com/ChainSafe/Multix/pull/239

### CODE REVIEW

**Comment by asnaith** [comment]
I nuked my local instance and started from scratch but I ran into this issue with `npm run codegen` :

```
➜  squid git:(tbaut-update-readme) npm run codegen;

> multix-squid@0.1.0 codegen
> npx squid-typeorm-codegen

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/squid-typeorm-codegen - Not found
npm ERR! 404
npm ERR! 404  'squid-typeorm-codegen@*' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in: /Users/asnaith/.npm/_logs/2023-07-11T21_51_33_837Z-debug-0.log
```

File: README.md [hunk 1]
```diff
@@ -24,39 +26,38 @@ BLOCK_START=3510000 # this is the starging block of the indexer
...
-
-# build the indexer, migrate the db and run the indexer
-# this will block your current terminal window
-yarn squid:start
+Install and launch the indexer and its DB. This will block the terminal
 
-# in a separate terminal window, run the graphql server
-yarn squid:start:graphql-server;
+```bash
+cd /squid;
```
Comment (Tbaut) [inline README.md:39]: npm install

Comment (Tbaut) [inline README.md:39]:
```suggestion
cd /squid;
npm install;
```

File: README.md [hunk 2]
```diff
@@ -24,39 +26,39 @@ BLOCK_START=3510000 # this is the starging block of the indexer
...
+npm install;
```
Comment (Tbaut) [inline README.md:40]:
```suggestion
cd squid;
npm run install;
```

**Comment by asnaith** [comment]
> @asnaith oh I forgot probably the most important part, the `npm run install` to install dependancies. Should be better now :)

...and I don't know how that didn't immediately occur to me either 😄

I had to make a small change but it's good to go now.

**Review by asnaith — APPROVED** [review]

**Comment by Tbaut** [comment]
Thanks!

**Pull request merged**